patches/server/0018-Multithreaded-entity-tracking.patch

```diff
@@ -297,7 +301,14 @@ index a42571cfd2c9c80df27e59db832cb64c2a64e141..e265a10781286a94d81086731ec0d333
          private final int trackingDistance;
          private SectionPosition e;
          // Paper start
-@@ -2483,7 +2503,7 @@ Sections go from 0..16. Now whenever a section is not empty, it can potentially
+         // Replace trackedPlayers Set with a Map. The value is true until the player receives
+         // their first update (which is forced to have absolute coordinates), false afterward.
+-        public java.util.Map<EntityPlayer, Boolean> trackedPlayerMap = new java.util.HashMap<>();
++        public java.util.Map<EntityPlayer, Boolean> trackedPlayerMap = new java.util.concurrent.ConcurrentHashMap<>(); // Airplane - concurrent EntityTracker access
```

Comment:
I think a concurrent map here is masking the real issue of the class being unable to handle concurrency, that needs to be fixed instead of using a bandaid.